Buildings/Controls/OBC/UnitConversions/From_BtuPerHour.mo

```diff
@@ -1,4 +1,5 @@
-block From_BtuPerh "Block that converts power from British thermal units per hour to watt"
+within Buildings.Controls.OBC.UnitConversions;
+block From_BtuPerHour "Block that converts power from British thermal units per hour to watt"
```

Comment:
Can you then please change it to `PerH`, e.g., use camel case rather than `Perh`.
Please do it on a new branch as this pull request fixes a bug (missing `within` statements) that causes OpenModelica to not load the library.

Reply:
How about Hr if camel case? H seems confusing.

Reply:
`Hr` would be ok, but is this still SI/ASHRAE? If not, we may as well use `Hour`